exercises/hello-world/HelloWorldSpec.groovy

```diff
@@ -0,0 +1,20 @@
+@Grab('org.spockframework:spock-core:1.0-groovy-2.4')
+import spock.lang.*
+
+class HelloWorldSpec extends Specification {
+
+    def 'outputs "Hello, World!"'() {
+        expect: new HelloWorld().hello() == 'Hello, World!'
```

Comment:
@gijsleussink I was also wondering about this one— you think that we should introduce `helloWorld` as a test case variable?

I could get behind that, but I was *almost* considering making the call `static`, since our calls just output text, and don't modify any object state, which would make the calls `HelloWorld.hello()`.